src/views/illness/illness-end-date.njk

```diff
@@ -79,7 +79,7 @@
               }
             },
             hint: {
-              text: 'For example, 31 8 2018'
+              text: 'You told us the illness started on ' + illnessStart | date
```

Comment:
I would add the logic in the controller. Something like

const someDate = `You told us the illness started on ${formatDate(illnessDate)}`;